### ISSUE TITLE
Render PDF fail due to header-includes

### ISSUE THREAD

**Issue by rgaiacs** [GESIS-Methods-Hub/andrew#187]
```
ERROR: Validation of YAML front matter failed.
ERROR: In file index.md
(line 16, column 3) Array entry 1 is empty but it must instead be a string.
15: header-includes:
16: - |
     ~
17:   <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>

ERROR: Render failed due to invalid YAML.
```

**Comment by rgaiacs** [comment]
YAML validation was fixed in Quarto v1.4.330.

**Comment by rgaiacs** [comment]
Should be fixed now that we avoid HTML.
